templates/footer.php

```diff
@@ -0,0 +1,7 @@
+<footer class="amp-wp-footer" role="contentinfo" itemscope="itemscope" itemtype="http://schema.org/WPFooter">
+	<div>
+		<h2><?php echo esc_html( $this->get( 'blog_name' ) ); ?></h2>
+		<p><?php _e( 'Powered by WordPress.com', 'amp' ); ?></p>
```

Comment:
Sure, I’ll change it to WordPress for now.

I’m not sure how we could connect it to the WP.com overrides, but I’ll ask Ernesto who worked on that new footer override script to see what we can do.